modules/backend/assets/js/october.tabformexpandcontrols.js

```diff
@@ -160,4 +160,114 @@
     }
 
     $.oc.tabFormExpandControls = TabFormExpandControls
-}(window.jQuery);
\ No newline at end of file
+
+}(window.jQuery);
+
+/*
+ * Auto update WAI-ARIA when a user updates the tabs
+ */
+ $(document).ready(function() {
+
+     /* If no selected => select first */
+     if ($('.master-tabs a[aria-selected]').length === 0) {
+         $('.master-tabs a:first').attr({
+             'aria-selected': 'true',
+             'tabindex': '0'
+         });
+     } else if ($('.primary-tabs a[aria-selected]').length === 0) {
+         $('.primary-tabs a:first').attr({
+             'aria-selected': 'true',
+             'tabindex': '0'
+         });
+     } else if ($('.secondary-tabs a[aria-selected]').length === 0) {
+         $('.secondary-tabs a:first').attr({
+             'aria-selected': 'true',
+             'tabindex': '0'
+         });
+     } else if ($('.content-tabs a[aria-selected]').length === 0) {
+         $('.content-tabs a:first').attr({
+             'aria-selected': 'true',
+             'tabindex': '0'
+         });
+     }
+     // else if ($('.vertical-tabs a[aria-selected]').length === 0) {
+     //     $('.vertical-tabs a:first').attr({'aria-selected':'true', 'tabindex':'0'});
+     //     $('.vertical-tabs ul').attr('aria-orientation':'vertical');
+     // }
+
+     /* Click on a link */
+     $('body').on('click', '.master-tabs,.primary-tabs,.secondary-tabs,.content-tabs', function(event) { // Can add ,.vertical-tabs
```

Comment:
@ayumihamsaki 
`$('body').on('click', '.master-tabs,.primary-tabs,.secondary-tabs,.content-tabs', function(event) {`
change to:
`$(''.master-tabs,.primary-tabs,.secondary-tabs,.content-tabs'').on('click', function(event) {`
You don't need to listen to body actions, if you really do, could you please add more info.

Reply:
@w20k that's to support tabs being dynamically added. Otherwise listeners are not added to elements that are added to the page through the AJAX framework.

Reply:
@LukeTowers, totally lost the thought, thanks for reminding ;)

Reply:
lol